packages/edit-site/src/components/left-sidebar/navigation-panel/content-navigation.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { useState } from '@wordpress/element';
+import {
+	__experimentalNavigation as Navigation,
+	__experimentalNavigationMenu as NavigationMenu,
+	__experimentalNavigationItem as NavigationItem,
+} from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ContentPagesMenu from './menus/content-pages';
+import ContentCategoriesMenu from './menus/content-categories';
+import ContentPostsMenu from './menus/content-posts';
+
+export default function ContentNavigation() {
+	const [ activeMenu, setActiveMenu ] = useState( 'root' );
```

Comment:
Having two `Navigation` components make things interesting. Should we store this in the store as well?

Reply:
cc @Copons on this since he was advertising the ability to have multiple navigations in one view 😄

Reply:
If it's working fine for now I'd defer the store migration for a separate PR.

Reply:
> advertising

😆
Well, I mean, the Navigation component is one part of the sidebar, so there are literally no reasons why a sidebar can't have more than one!

Also, IIRC the component's storybook used to have multiple `Navigation` in the same example (but this might have been just some WIP code on my machine, never released to the general public).